### PR TITLE
Change parameters

### PR DESCRIPTION
**Pull request by grzegorz-jakubiak**
Last thing related to https://www.pivotaltracker.com/story/show/165972475.

### CODE REVIEW

File: src/clients/asset-client.js [hunk 1]
```diff
@@ -133,19 +145,22 @@ class AssetClient {
...
   __createOrUpdate (options = {}) {
     const fileName = options.file.name
-    const variations = Object.keys(options.variations).map(key => {
-      return {
-        [key]: JSON.stringify(options.variations[key])
-      }
-    })
+    const formats = Object.keys(options.formats).reduce((acc, key) => {
+      const format = options.formats[key]
+      format.in = JSON.stringify(format.in)
```
Comment (jdahlke) [inline src/clients/asset-client.js]: `JSON.stringify` should not be necessary?

What about `format.ext`?

Comment (grzegorz-jakubiak) [reply src/clients/asset-client.js]: right I forgot it's stringified on `asset-service` side

Comment (grzegorz-jakubiak) [reply src/clients/asset-client.js]: >  What about `format.ext`?

You mean if it's not present?

Comment (jdahlke) [reply src/clients/asset-client.js]: No, sorry my fault for misunderstanding the code.

**Pull request merged**